lib/css/components/_stacks-inputs.less

```diff
@@ -108,6 +108,35 @@ fieldset {
     min-width: 0;
     padding: 0;
     border: 0;
+
+    &[disabled] {
+        a,
+        .s-btn,
+        .s-link {
+            box-shadow: none !important;
+            opacity: 0.5;
+            pointer-events: none;
+        }
+
+        .s-checkbox,
+        .s-input-message,
+        .s-label,
+        .s-radio,
+        .s-toggle-switch,
+        .s-toggle-switch label {
+            cursor: not-allowed;
+            opacity: 0.5;
+        }
+
+        .s-input,
+        .s-textarea,
+        .s-select > select {
+            border-color: var(--black-075);
+            background-color: var(--black-050);
+            color: var(--black-200);
+            cursor: not-allowed;
```

Comment:
Should we consider normalizing this around `opacity: 0.5` like we've done with other disabled states?

Reply:
Looks like I lifted this from https://github.com/StackExchange/Stacks/blob/develop/lib/css/components/_stacks-inputs.less#L399-L414

I tested out using opacity and it seems to work just fine. Can you think of any reason we would've used colors instead of opacity (just generally; not wrt fieldset)?

Reply:
Opacity felt like a shortcut at the time, but it's since been established as a common-practice.